src/pudl/extract/excel_test.py

```diff
@@ -34,8 +35,12 @@ def test_basics(self):
 class FakeExtractor(excel.GenericExtractor):
     """Test friendly fake extractor returns strings instead of files."""
 
-    METADATA = excel.Metadata('test')
-    BLACKLISTED_PAGES = ['shoes']
+    def __init__(self, *args, **kwargs):
+        """It's a Fake extractor.  Good thing flake demanded this."""
+        ds = datastore.Datastore(sandbox=True)
```

Comment:
Codacy found an issue: [No value for argument 'pudl_in' in constructor call](https://app.codacy.com/manual/zaneselvans/pudl/pullRequest?prid=6263807)